composer.json

```diff
@@ -18,34 +18,27 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "ext-openssl": "*",
```

Comment:
won't this need ext-openssl? Extensions are always a PITA to document

Reply:
Added `ext-openssl` again in https://github.com/pact-foundation/pact-php/pull/541